util/DataManager.js

```diff
@@ -0,0 +1,11 @@
+export default {
+    UpdateActivityList: (newList) => {
+        localStorage.setItem("activities", JSON.stringify(newList));
+    },
+    getActivityList: () => {
+        let localData = localStorage.getItem("activities");
+        let activities = JSON.parse(localData);
+        if (activities == null) activities = [];
```

Comment:
Dica pra economizar código

```suggestion
       return activities || []
```

Caso o primeiro operando (activies) seja um valor false (null, undefined, 0, '') ele retorna o segundo valor, caso contrario retorna o primeiro mesmo:

![image](https://user-images.githubusercontent.com/15300776/119402305-22909280-bcb3-11eb-9ef4-cbf0f8f44aa3.png)